src/translations/nbTranslations.json

```diff
@@ -2236,6 +2233,9 @@
     "user_dialog": "Brukerdialog",
     "your_tasks": "Oppgaver du behandler: {{count}}"
   },
+  "no_search_results_found_with_search": "<span><icon /><heading>Ingen treff</heading></span><p>Vi fant dessverre ikke noe som passer med søket ditt.</p>",
```

Comment:
kanskje litt mystisk hva `<span>` er her, spesielt for oversetter, så kunne vurdert å droppa den til fordel for `ml: '0.5rem'` på heading også

Reply:
aha såklart, da må den nok med ja